.gitignore

```diff
@@ -1,8 +1,19 @@
+.gitignore
```

Comment:
I'm not sure I want this behavior. If something's being added to the gitignore of a subfolder (i.e. `jxlint-impl/.gitignore`), it should probably be committed to the repo. If that doesn't work, you can add it to `.git/info/exclude` [as described here](https://stackoverflow.com/a/767213/536910).